examples/windows/dll/hello-library.h

```diff
@@ -0,0 +1,13 @@
+#ifndef HELLO_LIBRARY_H
+#define HELLO_LIBRARY_H
+
+#if COMPILING_DLL
```

Comment:
nit: `#ifdef`

Reply:
Thanks! Addressed in the internal change.